app/assets/stylesheets/modules/forms.scss

```diff
@@ -7,3 +7,8 @@
     text-align: center;
   }
 }
+
+.form-group h2,
+.searchworks-item {
+  margin: 0 0 1em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 1em` instead of `0 0 1em 0`